rules/how-to-decide-what-to-test/rule.md

```diff
@@ -0,0 +1,66 @@
+
+Under such conditions of uncertainty (which are normal in software development), look for methods or ways of coming up with test ideas that might work, while acknowledging that they might not - these are **heuristics**.
+
+### Heuristics
+
+::: greybox
+A heuristic is a fallible method for solving a problem or making a decision
+
+A heuristic is an experience-based technique for problem solving, learning and discovery
+:::
+
+> A heuristic is a way to help me come up with test ideas - Lee Hawkins
+
+
+You'll likely build up your own toolbox of heuristics to draw from as you become more familiar with them and realise their power. 
+
+### Getting started - consistency heuristics
+
+
+To get started with the use of heuristics, try some **consistency heuristics**. A great example comes from Michael Bolton in the form of [HICCUPPS](https://developsense.com/blog/2012/07/few-hiccupps), which is a mnemonic as follows:
+
+* **H**istory. We expect the present version of the system to be consistent with past versions of it.
+
+* **I**mage. We expect the system to be consistent with an image that the organization wants to project, with its brand or with its reputation.
+* **C**omparable Products. We expect the system to be consistent with systems that are in some way comparable. This includes other products in the same product line; competitive products, services, or systems; or products that are not in the same category but which process the same data; or alternative processes or algorithms.
+* **C**laims. We expect the system to be consistent with things important people say about it, whether in writing (references, specifications, design documents, manuals, whiteboard sketches, etc.) or in conversation (meetings, public announcements, lunchroom conversations, etc.).
+* **U**sers' Desires. We believe that the system should be consistent with ideas about what reasonable users might want. 
+* **P**roduct. We expect each element of the system (or product) to be consistent with comparable elements in the same system.
+* **P**urpose. We expect the system to be consistent with the explicit and implicit uses to which people might put it.
+* **S**tatutes. We expect a system to be consistent with laws or regulations that are relevant to the product or its use.
+
+As an example of using the "Product" part of this heuristic, note that in Trello, there is an inconsistency in the way fields on cards can be edited. Clicking in the Description or Tasks fields puts the field straight into Edit mode, whereas to edit a Comment requires an explicit click on the "Edit" action. This is a case of the Trello product being inconsistent with itself in terms of editing fields on a card.
+
+### Further reading
+
+[Cultivate Your Credibility With Oracles And Heuristics](https://www.ministryoftesting.com/dojo/lessons/cultivate-your-credibility-with-oracles-and-heuristics) by Lee Hawkins
```

Comment:
I would make this as a unordered list

Reply:
For a single item? Happy for you to change it if that's the standard, but feels a bit odd to me

Reply:
> For a single item? Happy for you to change it if that's the standard, but feels a bit odd to me

Yes, for consistency. I have made that change.

BTW - I wouldn't capitalize "Your", "With", and "And" as per https://www.grammarly.com/blog/capitalization-in-the-titles/ 
Thoughts?

Reply:
I try to match the name exactly with whatever I'm linking to, as I've done in this case. In naming my own stuff, I follow something like the Grammarly recommendation.

Reply:
I see what you mean, but I think we should fix it anyways, so I've update the file. Won't make a difference to the reference but will for us (consistency)